msteams-platform/bots/how-to/authentication/bot-sso-manifest.md

```diff
@@ -0,0 +1,210 @@
+      "entityId": "simpleAuth",
+      "name": "Simple Auth",
+      "scopes": [
+        "personal"
+      ]
+    },
+    {
+      "contentUrl": "https://<<BASE_URI_DOMAIN>>/tab/silent?loginHint={loginHint}&userObjectId={userObjectId}&tenantId={tid}",
+      "entityId": "silentAuth",
+      "name": "Silent Auth",
+      "scopes": [
+        "personal"
+      ]
+    },
+    {
+      "contentUrl": "https://<<BASE_URI_DOMAIN>>/tab/sso",
+      "entityId": "ssoAuth",
+      "name": "SSO Auth",
+      "scopes": [
+        "personal"
+      ]
+    }
+  ],
+  "bots": [
+    {
+      "botId": "<<REGISTERED_BOT_ID>>",
+      "scopes": [
+        "personal"
+      ]
+    }
+  ],
+  "permissions": [
+    "messageTeamMembers",
+    "identity"
+  ],
+  "validDomains": [
+    "<<BASE_URI_DOMAIN>>",
+    "token.botframework.com"
+  ],
+  "webApplicationInfo": {
+      "id": "<<REGISTERED_BOT_ID>>",
+      "resource": "api://<<BASE_URI_DOMAIN>>/<<REGISTERED_BOT_ID>>"
+  }
+}
+```
+
+</details>
+
+> [!NOTE]
+> During debug, you can use ngrok to test your app in Azure AD. In that case, you need to replace the subdomain in `api://subdomain.example.com/00000000-0000-0000-0000-000000000000` with the ngrok URL. You'll need to update the URL whenever your ngrok subdomain changes. For example, api://23c3-103-50-148-128.ngrok.io/bccfbe67-e08b-4ec1-a7fd-e0aaf41a097c.
+
+## Sideload and Preview in Teams
+
+You've configured the app to enable SSO. You can now sideload your app in Teams, and preview it in Teams environment.
+
+To preview your app in Teams:
+
+1. Create an app package.
+
+   The app package is a zip file that contains the app manifest file and app icons.
+
+1. Open Teams.
+
+1. Select **Apps** > **Manage your apps** > **Upload an app**.
+
+    The options to upload an app appear.
+
+1. Select **Upload a custom app** to sideload the app into Teams.
+
+1. Select your app package zip file, and then select **Add**.
+
+    The app is sideloaded. The consent dialog appears to inform you of the permissions that may be required.
+
+1. Select **Continue**.
+
+    :::image type="content" source="../../../assets/images/authentication/teams-sso-bots/bot-consent.png" alt-text="Bot consent message" border="false":::
+
+  Teams opens the app and you can use it.
+
+  Congratulations! You've enabled SSO for your bot or message extension app.
+
+## SSO support in Developer Portal
+
+You can also upload your SSO-enabled app in Developer Portal. To enable SSO support in Developer Portal after you've imported it:
+
+1. Open Developer Portal.
+1. Select **Tools**, and from the list of apps that appears select your app to open it in Developer Portal.
+1. Select **Configure**.
+
+    The **Configure** page appears.
+
+1. Enter the messaging endpoint you configured in Azure AD as **Endpoint address**.
+
+    :::image type="content" source="../../../assets/images/authentication/teams-sso-bots/dev-portal-sso.png" alt-text="Developer Portal support for SSO in a bot app":::
```

Comment:
the preview doesnt seem to render image, is this expected?

Reply:
Hello @pradeepananth, Thank you for the comment. The images are rendering for me, and other who are reviewing the PR. Please let me know if any error is showing up instead of the image.